README-HE.md

```diff
@@ -0,0 +1,192 @@
+- [Git-it](https://github.com/jlord/git-it-electron) - מדריך גיט לאפליקצית הדסקטופ.
+- [Flight Rules לגיט](https://github.com/k88hudson/git-flight-rules) - מדריך על מה לעשות כשדברים משתבשים.
+- [מדריך גיט בספרדית למתחילים](https://platzi.github.io/git-slides/#/) - מדריך שלם של שקופיות על גיט וגיטהאב בספרדית. Una guía completa de diapositivas sobre git y GitHub explicadas en Español.
+- [Git Kraken](https://www.gitkraken.com/git-client) - אפליקצית דסקטופ לגיט, ויז׳ואלית, אינטרקטיבית ו-cross-platform.
+- [טיפים לגיט](https://github.com/git-tips/tips) - אוסף טיפים וטריקים נפוצים בגיט.
+- [שיטות עבודה מומלצות בגיט](https://sethrobertson.github.io/GitBestPractices/) - בצעו commit לעיתים קרובות, שפרו בהמשך, פרסמו פעם אחת: שיטות העבודה המומלצות בגיט.
+- [מדריך גיט אינטרקטיבי](https://learngitbranching.js.org/) - למדו גיט בדרך הויז׳יאלית והאינטרקטיבית ביותר.
+
+</p>
+</div>
+<div dir="rtl" style="direction:rtl;text-align:right;">
+<p>
+
+## ספרים בנושא קוד פתוח
+- [יצירת תוכנה בקוד פתוח](https://producingoss.com/) - זהו ספר על הצד האנושי של פיתוח קוד פתוח. הספר מתאר כיצד פרויקטים מוצלחים עובדים, ציפיות האנשים והמפתחים ותרבות תכונה חינמית.
+- [סדרת ספרי קוד פתוח](https://opensource.com/resources/ebooks) - למדו עוד על קוד פתוח ועל תנועת הקוד הפתוח בעזרת רשימה מקיפה של ספרים אלקטרונים מ-https://opensource.com.
+- [Software Release Practice HOWTO](https://en.tldp.org/HOWTO/Software-Release-Practice-HOWTO/) - מדריך המתאר מהם הכללים לשחרור נכון עבור לינוקס ופרויקטי קוד פתוח אחרים. על ידי שמירה על כללים אלו, תקלו על משתמשים לבנות את הקוד שלכם ולהשתמש בו. כמו כן, מפתחים אחרים יוכלו להבין את הקוד שלכם, לעבוד איתו ולשפר אותו. 
+- [קוד פתוח 2.0: אבולוציה מתמשכת](https://archive.org/details/opensources2.000diborich) (2005) - קוד פתוח 2.0 הינו אוסף מאמרים מעמיקים ומעוררי מחשבה של מנהיגי הטכנולוגיה של היום הממשיכים לצבוע את תמונת האבולוציה שפותחה בספר מ-1999, קוד פתוח: קולות מן המהפכה.  
+- [הארכיטקטורה של אפליקציות קוד פתוח](http://www.aosabook.org/en/git.html) - מגוון היבטים המתארים כיצד גיט עובד מתחת למנוע כדי לאפשר workflows מבוזרים וכיצד אלו שונים ממערכות version controls אחרים (VCSs).
+- [קודים פתוחים: קולות ממהפכת הקוד הפתוח](https://www.oreilly.com/openbook/opensources/book/) - מאמרים מחלוצי הקוד הפתוח כמו Linus Torvalds (לינוקס), Larry Wall (פרל) ו-Richard Stallman (GNU).
+
+</p>
+</div>
+<div dir="rtl" style="direction:rtl;text-align:right;">
+<p>
+
+## יוזמות לתרומת קוד פתוח
+- [Up For Grabs](https://up-for-grabs.net/) - מכיל פרויקטים עם issues המתאימים למתחילים.
+- [First Timers Only](https://www.firsttimersonly.com/) - רשימה של באגים המתוייגים כ-״first-timers-only".
+- [First Contributions](https://firstcontributions.github.io/) - בצעו את התרומה הראשונה שלכם לקוד פתוח ב-5 דקות. כלי ומדריך שיעזור למתחילים לתרום. תוכלו למצוא את קוד המקור של האתר [כאן](https://github.com/firstcontributions/first-contributions), נצלו את ההזדמנות לתרום ל-repository עצמו.
```

Comment:
I think that maybe we should change `ב-5 דקות`  to `תוך 5 דקות`
To make it appear more official

Reply:
It does sound better, fixed!